src/print-prescription/prescription-printout.component.tsx

```diff
@@ -0,0 +1,119 @@
+import { Layer, StructuredListBody, StructuredListCell, StructuredListRow, StructuredListWrapper } from '@carbon/react';
+import { formatDate, parseDate, useSession } from '@openmrs/esm-framework';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { type DosageInstruction, type MedicationRequestBundle, type Quantity } from '../types';
+import {
+  getDosageInstruction,
+  getMedicationDisplay,
+  getMedicationReferenceOrCodeableConcept,
+  getQuantity,
+  getRefillsAllowed,
+} from '../utils';
+import styles from './print-prescription.scss';
+
+type PrescriptionsPrintoutProps = {
+  medicationrequests: Array<MedicationRequestBundle>;
+  excludedPrescription: Array<string>;
+};
+
+const PrescriptionsPrintout: React.FC<PrescriptionsPrintoutProps> = ({ medicationrequests, excludedPrescription }) => {
+  const { t } = useTranslation();
+  const {
+    sessionLocation: { display: facilityName },
+  } = useSession();
+  const patient = medicationrequests[0]?.request?.subject;
+  const extractpatientName = (display: string) => display.match(/^([A-Za-z\s]+)\s\(/)?.at(1);
+  return (
+    <Layer className={styles.printOutContainer}>
+      <StructuredListWrapper>
+        <StructuredListBody>
+          <StructuredListRow head>
+            <StructuredListCell head>
+              <br />
+              <br />
+              <p className={styles.printoutTitle}>{t('prescriptionInstructions', 'Prescription Instructions')}</p>
+              {patient && (
+                <p className={styles.faintText} style={{ textAlign: 'center' }}>
+                  {extractpatientName(patient.display)?.toUpperCase()}
```

Comment:
@denniskigen  @ibacher @chibongho  do we have an existing utility method we could use here instead?

Reply:
No, but this `extractpatientName()` method is definitely not it (this regex excludes any accent marks, any non-Latin scripts, which means it does not work in a language-agnostic way). A much better (and simpler) way of doing the exact same thing is this:

```ts
display.includes('(') ? display.split('(')[0] : display;
```

We should probably add something like this to the framework utilities, though.

I'm also not hugely a fan of having the call to `toUpperCase()` here. A CSS text-transform would be much cleaner here.

Reply:
One last thing, the mixing of both `className` and `style` on a single tag should not be done. Just add a second class.

Reply:
@Omoshlawi Did you see the above comments?